desktop/apps/auction/routes.js

```diff
@@ -37,22 +37,14 @@ export async function index (req, res, next) {
   const auctionArticles = new Articles(articles)
 
   try {
-    const html = makeTemplate('index.jade', {
-      basePath: res.app.get('views'),
-      locals: {
-        ...res.locals,
```

Comment:
@craigspaeth - this is the issue right here.

Reply:
Going to leave [this link](https://github.com/artsy/stitch/pull/84#discussion_r332930158) here for posterity to a write-up of me looking into this and being fairly certain that it _is_ due to data being shared incorrectly (probably by adding personalised data to the `sharify.data` singleton) and not the template caching.